tests/src/python/test_provider_sensorthings.py

```diff
@@ -1333,6 +1333,228 @@ def test_subset_string(self):
                                                        1.62337299999999995,
                                                        52.13201699999999761))
 
+    def test_feature_limit(self):
+        with tempfile.TemporaryDirectory() as temp_dir:
+            base_path = temp_dir.replace("\\", "/")
+            endpoint = base_path + "/fake_qgis_http_endpoint"
+            with open(sanitize(endpoint, ""), "wt", encoding="utf8") as f:
```

Comment:
can you  put a comment on each block to "explain" what you like to test?
```suggestion
           # check if we retrieve the endpoint list
            with open(sanitize(endpoint, ""), "wt", encoding="utf8") as f:
```

Reply:
No problem! done